default-configs/i386-softmmu.mak

```diff
@@ -25,3 +25,4 @@
 CONFIG_ISAPC=y
```

Comment:
Ideally these should be able to be disabled and only `CONFIG_XBOX` should need to be enabled, but there are some dependencies here that I haven't eliminated yet. Leaving as-is for now.

Reply:
We should probably create an issue after merge.

It's probably a bit confusing to have a dedicated `XBOX` option that still depends on other options (when Kconfig is meant to take care of this).